code/modules/cm_tech/techs/marine/tier3/cryo_spec.dm

```diff
@@ -0,0 +1,24 @@
+/datum/tech/cryomarine
+	name = "Wake Up Additional Specialist"
+	desc = "Wakes up an additional specialist to fight against any threats."
+	icon_state = "overshield"
+
+	announce_name = "ALMAYER SPECIAL ASSETS AUTHORIZED"
+	announce_message = "An additional specialist is being taken out of cryo."
+
+	required_points = 8
+
+	flags = TREE_FLAG_MARINE
+	tier = /datum/tier/three
+
+/datum/tech/cryomarine/can_unlock(mob/M)
+	. = ..()
+	if(!.)
+		return
+	if(!SSticker.mode)
+		to_chat(M, SPAN_WARNING("You can't do this right now!"))
```

Comment:
```suggestion
/datum/tech/cryomarine/can_unlock(mob/user)
	. = ..()
	if(!.)
		return
	if(!SSticker.mode)
		to_chat(user, SPAN_WARNING("You can't do this right now!"))
```

Reply:
also why isnt this check just on /datum/tech/proc/can_unlock i dont understand anything